### DISCUSSION TITLE
EVID will be only coded as 1 for the first dose and will be 0 for all additional doses

### DISCUSSION THREAD

**Discussion by mahmed452** [metrumresearchgroup/mrgsolve#847] [Q&A]
I am simulating dataset where i am giving 6 doses of 300 mg every 4 weeks. i asked for evid in the output, but it will only show it as 1 for the first dose

sim=mod %>% data_set(data)%>%
  zero_re() %>%
  mrgsim(end=4000,recover=c('regimen','Group','WT'),carry_out='evid',delta=672) %>%  as_tibble()
head(sim)

# A tibble: 14 x 9
      ID  time  evid   GUT  CENT    CP regimen     Group    WT
   <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <chr>       <chr> <dbl>
 1     1     0     0    0    0    0    A: 300 q 4w A      105.
 2     1     0     1  300    0    0    A: 300 q 4w A      105.
 3     1   672     0  300.  84.1  4.92 A: 300 q 4w A      105.
 4     1  1344     0  300. 105.   6.13 A: 300 q 4w A      105.
 5     1  2016     0  300. 110.   6.43 A: 300 q 4w A      105.
 6     1  2688     0  300. 111.   6.50 A: 300 q 4w A      105.
 7     1  3360     0  300. 111.   6.52 A: 300 q 4w A      105.
 8     2     0     0    0    0    0    A: 300 q 4w A      122.
 9     2     0     1  300    0    0    A: 300 q 4w A      122.
10     2   672     0  300.  81.3  4.26 A: 300 q 4w A      122.
11     2  1344     0  300. 101.   5.27 A: 300 q 4w A      122.
12     2  2016     0  300. 105.   5.51 A: 300 q 4w A      122.
13     2  2688     0  300. 106.   5.57 A: 300 q 4w A      122.
14     2  3360     0  300. 106.   5.58 A: 300 q 4w A      122.

**Comment by kylebaron** [comment]
@mahmed452 Did you figure this out?  If the additional doses are scheduled via `addl` in the data set, then you won't see those records in the output. You have to explicitly included those in the data. There's a helper function called `realize_addl()` that could help, but this is a difficult thing when there for example time-varying covariates.

**Comment by mahmed452** [comment]
Thanks Kyle. I didn't know about this function so I explicitly added these
records in my dataset. But good to know about this function.
